repose-aggregator/components/filters/herp-filter/src/main/scala/org/openrepose/filters/herp/HerpFilter.scala

```diff
@@ -121,8 +123,17 @@ class HerpFilter @Inject()(configurationService: ConfigurationService,
       else stripHeaderParams(allProjectIds.maxBy(getQuality))
     }
 
-    val projectIds = httpServletRequest.getHeaders(OpenStackServiceHeader.TENANT_ID.toString).asScala
-      .++(httpServletRequest.getHeaders(X_PROJECT_ID).asScala).toTraversable
+    def getSplitHeaders(headerInteractor: HeaderInteractor, headers: List[String]): Traversable[String] = {
+      headers.foldLeft(Traversable[String]()) { (accumulator, current) =>
```

Comment:
I'm not sure if we have a real convention, but I've always preferred the `Traversable.empty[String]` style. It just seems clearer to me (since you avoid the parenthesis pile-up).